README.md

```diff
@@ -385,6 +435,58 @@ value = vals->get(0);
 
 Using raw pointer based API may improve performance if you are using the API to return multiple values such as values, keySet, and entrySet. In this case, cost of deserialization is delayed until the item is actually accessed.
 
+# Mixed Object Types Supporting HazelcastClient
+Sometimes, you may need to use Hazelcast data structures with objects of different types. E.g. you may want to put int, string,identifieddataserializable, etc... objects into the same Hazelcast IMap data structure. You can do this by using the mixed type adopted HazelcastClient. You can adopt the client in this way:
+```
+    ClientConfig config;
+    HazelcastClient client(config);
+    mixedtype::HazelcastClient &hazelcastClient = client.toMixedType();
+``` 
+The mixedtype::HazelcastClient interface is designed to provide you the data structures which allows you to work with any object types in a mixed manner. E.g. The interface allows you to provide the key and value type differently for each map.put call. An example usage:
+```
+    mixedtype::IMap map = hazelcastClient.getMap("MyMap");
+
+    map.put<int, int>(3, 5);
+    map.put<string, std::string>("string key1", "MyStringValue");
+    map.put<int, MyCustomObject>("string key1", myCustomInstance);
+    TypedData result = map.get<int>(3);
+``` 
+
+As you can see in the above code snippet, we are putting int, string and MyCustomObject to the same map. Both the key and value can be different type for each map.put call.
```

Comment:
...value can be of different type for.....

Reply:
ok, done.